app/src/jembi/res/layout/about.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingLeft="24dp"
+        android:paddingRight="24dp"
+        android:paddingTop="4dp"
+        android:orientation="vertical">
+
+        <TextView
+            android:text="@string/about_version"
+            style="@style/CaptionText" />
+        <TextView
+            android:id="@+id/version"
+            android:text="@string/unknown"
+            style="@style/BodyText" />
+
+        <TextView
+            android:text="@string/about_site"
+            style="@style/CaptionText" />
+        <TextView
+            android:text="@string/site_link"
+            android:autoLink="web"
+            style="@style/BodyText" />
+
+        <TextView
+            android:text="@string/about_mail"
+            style="@style/CaptionText" />
+        <TextView
+            android:text="@string/team_email"
+            android:autoLink="email"
+            style="@style/BodyText" />
+
+        <TextView
+            android:text="@string/about_source"
+            style="@style/CaptionText" />
+        <TextView
+            android:text="@string/source_link"
+            android:autoLink="web"
+            style="@style/BodyText" />
+
+        <TextView
+            android:text="@string/about_license"
+            style="@style/CaptionText" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:gravity="center"
+            android:layout_marginTop="12dp"
+            android:paddingBottom="16dp"
+            android:clipToPadding="false">
+
+            <Button
+                android:layout_width="wrap_content"
+                android:minWidth="100dp"
+                android:layout_height="wrap_content"
+                style="@style/DetailsPrimaryButtonStyle"
+                android:id="@+id/ok_button"
+                android:text="@string/ok" />
+
+        </LinearLayout>
+
+    </LinearLayout>
+
+</ScrollView>
```

Comment:
@patience-mpofu I compared this file with the original, and don't see any changes. I believe this is because all the text for the page comes from strings, so just changing the strings should be enough and this copy can be removed.

Reply:
Removed here https://github.com/jembi/fdroidclient/commit/02aa092e5307e35ada23b824c3d79f76df32863d